roles/common/tasks/ssh.yml

```diff
@@ -9,12 +9,15 @@
     state: directory
 
 - name: Set authorized keys
-  template:
-    src: templates/ssh/{{ inventory_hostname }}/authorized_keys.j2
-    dest: /root/.ssh/authorized_keys
-    owner: root
-    mode: 0600
+  authorized_key:
```

Comment:
From my understanding of the [authorized_key](https://docs.ansible.com/ansible/authorized_key_module.html) module, it simply does a `presence` check for each key